src/java.base/share/classes/sun/net/www/http/HttpClient.java

```diff
@@ -144,12 +144,27 @@ private static int getDefaultPort(String proto) {
     // Traffic capture tool, if configured. See HttpCapture class for info
     private HttpCapture capture = null;
 
+    /* "jdk.https.negotiate.cbt" property can be set to "always" (always sent), "never" (never sent) or
+     * "domain:a,c.d,*.e.f" (sent to host a, or c.d or to the domain e.f and any of its subdomains). This is
+     * a comma separated list of arbitrary length with no white-space allowed.
+     * If enabled (for a particular destination) then SPNEGO authentication requests will include
```

Comment:
Previously `Negotiate` was used, now `SPNEGO`?

Reply:
"Negotiate" is the name of the HTTP authentication scheme which uses the SPNEGO mechanism. Possibly makes more sense to refer to Negotiate here.

Reply:
Yes, I know. That's the point. Clearly differentiate between GSS-API mech and HTTP auth scheme.